.azure/scripts/create-commits-history.ps1

```diff
@@ -0,0 +1,94 @@
+param (
+    [string]$Token
+)
+
+$ErrorActionPreference = 'Stop'
+
+git clone https://github.com/SSWConsulting/SSW.Rules.Content.git
+
+cd SSW.Rules.Content/
+
+#Step 1: Fetch all contributors - Retrieve from GitHub
+$authors = @()
+$apiUrl = "https://api.github.com/repos/SSWConsulting/SSW.Rules.Content/contributors?per_page=100&page="
+
+for ($page = 1; $page -le 2; $page++) {
+    $url = $apiUrl + $page
+    $headers = @{
+        "Authorization" = "Bearer $Token"
+    }
+    $response = Invoke-RestMethod -Uri $url -Method Get
+    
+    foreach ($contributor in $response) {
+        $authors += $contributor.login
+    }
+}
```

Comment:
Use proper pagination as per github docs
https://docs.github.com/en/rest/guides/using-pagination-in-the-rest-api?apiVersion=2022-11-28